client/packages/system/src/Patient/api/operations.graphql

```diff
@@ -1,6 +1,5 @@
 fragment PatientRow on PatientNode {
   id
-  code
```

Comment:
oh. hasn't made it back in yet? The code is important for current clients

Reply:
Thanks, forgot to put it back. Done now.